core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -168,7 +168,7 @@ public int run(final String[] args,
             consumerConfig.putAll(properties);
             exitCode = maybeResetInputAndSeekToEndIntermediateTopicOffsets(consumerConfig, dryRun);
             maybeDeleteInternalTopics(adminClient, dryRun);
-
+            System.out.println("succeed to reset stream application: " + groupId);
```

Comment:
super nit: we already print `Done` upon successful reset of the application (at the end of `#maybeResetInputAndSeekToEndIntermediateTopicOffsets`), can we just reuse that in the system test so we don't have to add any new print statements?

Reply:
copy that